tensorflow_recommenders_addons/dynamic_embedding/core/kernels/cuckoo_hashtable_op.cc

```diff
@@ -304,6 +304,18 @@ class CuckooHashTableOfTensors final : public LookupInterface {
     return table_->export_values(ctx, value_dim);
   }
 
+  Status Save(OpKernelContext* ctx, const string filepath,
+              const size_t buffer_size) {
+    int64 value_dim = value_shape_.dim_size(0);
```

Comment:
int64_t

Reply:
tensorflow::int64 is returned from `dim_size` and also used in table_->save()